src/features/program/types/strapiProgram.ts

```diff
@@ -0,0 +1,92 @@
+export type Params = {
+  method?: string
+  branch?: string
+  language?: string
+  category?: string
+  programCategory: string
```

Comment:
```suggestion
  method?: string
  branch?: string
  language?: string
  category?: string
  programCategory: string
```
ini bukannya bisa dipake enum jg ya kaya di program options? ambil yang dari program options aja sama kan ya

Reply:
iyaa deng, gw adjust dlu